internal/pkg/cli/job_package.go

```diff
@@ -57,6 +73,38 @@ func newPackageJobOpts(vars packageJobVars) (*packageJobOpts, error) {
 		sel:            selector.NewWorkspaceSelect(prompter, store, ws),
 		prompt:         prompter,
 	}
+
+	opts.stackSerializer = func(mft interface{}, env *config.Environment, app *config.Application, rc stack.RuntimeConfig) (stackSerializer, error) {
+		var serializer stackSerializer
+		jobMft := mft.(*manifest.ScheduledJob)
+		serializer, err := stack.NewScheduledJob(jobMft, env.Name, app.Name, rc)
+		if err != nil {
+			return nil, fmt.Errorf("init scheduled job stack serializer: %w", err)
+		}
+		return serializer, nil
+	}
+
+	opts.newPackageCmd = func(jobOpts *packageJobOpts) error {
```

Comment:
nit: error return not needed

Reply:
Tiny nit: we could just name it to be `o` here since the variable scope is pretty small.